students.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="studentstyle.css" media="screen" />
+    <title>T4G Research Group</title>
+  </head>
+  
+  
+  <body>
+
+    <h1>Alumni</h1>
+    <h3>Description for alumni page </h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ac metus elit. Suspendisse pellentesque tellus imperdiet, faucibus tellus at, mattis nisl.
+      Aliquam vestibulum augue vitae libero rhoncus, ac suscipit sem venenatis. Curabitur eu ligula auctor, pellentesque metus nec, venenatis mi. Proin congue nibh sed tellus luctus, 
+      sed mattis mi dictum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ac metus elit. Suspendisse pellentesque tellus imperdiet, faucibus tellus at, mattis nisl.
+      Aliquam vestibulum augue vitae libero rhoncus, ac suscipit sem venenatis. Curabitur eu ligula auctor, pellentesque metus nec, venenatis mi. Proin congue nibh sed tellus luctus, 
+      sed mattis mi dictum.</p>
+    
+    <div clas="alumni-list">
+      <div class="container-student">
+        <div class="card">
+          <img class="profile-picture" src="./img/profile-picture.jpg">
+          <div class="container-student-info">
+            <h2>STUDENT NAME</h2>
+            <p id="student-bio">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ac metus elit. Suspendisse pellentesque tellus imperdiet, faucibus tellus at, mattis nisl.
+              Aliquam vestibulum augue vitae libero rhoncus, ac suscipit sem venenatis. Curabitur eu ligula auctor, pellentesque metus nec, venenatis mi. Proin congue nibh sed tellus luctus, 
+              sed mattis mi dictum.
+            </p>
+            <div class="contact-box">
+              <!---
+              More info goes here
+  
+              -->
+              <p id="contact-alignment">Email: <a href="mailto: husky@uw.edu" style="text-decoration: none;">husky@uw.edu</a> <br> 
```

Comment:
I'd consider moving these `style` attributes out into the `studentstyle.css` file. Makes it easier to manage if all of the styling for the page is in one place.

Reply:
agree! styles for <a> are moved to ``studentstyle.css``